### ISSUE TITLE
Compile Mini Gunrock with ptx assemble issue

### ISSUE THREAD

**Issue by YuxinxinChen** [gunrock/mini#2]
I tried compile Mini Gunrock bfs and got following issue:
```
ptxas /tmp/tmpxft_00009acd_00000000-5_test_bfs.ptx, line 8902; error   : Instruction 'shfl' without '.sync' is not supported on .target sm_70 and higher from PTX ISA version 6.4
ptxas /tmp/tmpxft_00009acd_00000000-5_test_bfs.ptx, line 8906; error   : Instruction 'shfl' without '.sync' is not supported on .target sm_70 and higher from PTX ISA version 6.4
ptxas fatal   : Ptx assembly aborted due to errors
Makefile:18: recipe for target 'bin/test_bfs__x86_64' failed
make: *** [bin/test_bfs__x86_64] Error 255
```
I compiled with CUDA10 on daisy V100:
```
nvcc: NVIDIA (R) Cuda compiler driver
Copyright (c) 2005-2019 NVIDIA Corporation
Built on Wed_Apr_24_19:10:27_PDT_2019
Cuda compilation tools, release 10.1, V10.1.168
```
I suspect this is the moderngpu problem.

Thanks!
Yuxin

**Comment by yzhwang** [comment]
Pending on this PR:
https://github.com/moderngpu/moderngpu/pull/35

I will update upstream of submodule after that's been merged in. If you want to use mini now you can replace the whole moderngpu dir with current head of:
https://github.com/yzhwang/moderngpu

**Comment by yzhwang** [comment]
With the above mentioned PR merged into moderngpu, I've updated the submodule. Now the current head of mini should compile with CUDA 10+. Please confirm and close the issue if it is resolved.

**Comment by YuxinxinChen** [comment]
It can be compiled successfully now! Thanks!! @yzhwang
